roles/confluent.common/meta/main.yml

```diff
@@ -1,3 +0,0 @@
----
```

Comment:
I forget if this is a github thing, but shouldnt we see these files as moved, and a line changed... i guess its not a big deal. this file does not get updated ever so merge conflicts wont really happen

Reply:
It IS a GitHub thing, when you look at the total of the PR. But it you look at the commits, the history is preserved:

````
D:~/projects/github/confluent-cp-ansible (70-collections) $ git --no-pager log --follow roles/common/meta/main.yml
commit 3ac2b95ee3c67abee51ecfde19563c24351797bc
Author: Erik Godding Boye <egboye@gmail.com>
Date:   Thu Apr 1 21:30:37 2021 +0200

    Rename confluent.common role

commit def2fa498f89e2d50024d50efcb286fd9d09493c
Author: Erik Godding Boye <erik.boye@statnett.no>
Date:   Thu Apr 1 11:09:29 2021 +0200

    Rename confluent.variables role

commit ecbe0293a7c8cc2e1146ddb404edcc8df22d442e
Author: Domenic Bove <domenicbove@gmail.com>
Date:   Thu Jun 25 10:05:17 2020 -0700

    renamed confluent.variables_handlers to confluent.variables

commit 75c47ab41a5e0e0e8d8c54dfaca167cac37c5dbe
Author: Domenic Bove <domenicbove@gmail.com>
Date:   Mon Sep 16 10:19:13 2019 -0700

    renamed confluent.variables to confluent.variables_handlers

commit 45cb20f0a64d66ee666c4e0b98405146ff3a80cf
Author: Domenic Bove <domenicbove@gmail.com>
Date:   Fri Sep 13 11:53:36 2019 -0700

    default var for jolokia enabled created passed to all roles, need to build logic around empty strings

commit 230cd424caa4c2bd321bb637082b11cc986d49df
Author: Domenic Bove <domenicbove@gmail.com>
Date:   Tue Sep 3 14:39:11 2019 -0700

    roles now depend on vars role
D:~/projects/github/confluent-cp-ansible (70-collections) $
````

So I am quite sure this is not a problem.